fsharp/falco/web.fsproj

```diff
@@ -3,10 +3,11 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="App.fs" />
     <Compile Include="Program.fs" />
   </ItemGroup>  
   <ItemGroup>
     <PackageReference Update="FSharp.Core" Version="5.0.0" />
-    <PackageReference Include="Falco" Version="2.0.*" />
+    <PackageReference Include="Falco" Version="2.1.0" />
```

Comment:
should we not use `2.1.*` ?

Reply:
Apologies. I thought you were requesting me to remove the floating minor version.

Reply:
up to you ... if you do not plan to add **breaking changes** in patch level 😛